new-frontiers-14/frontier-station-14
# Power Cell/Svalinn Laser Pistol Projectile Change

## About the PR
Changes the projectile type of power cells to RedLaser [which deals 14 damage] instead of RedLightLaser [which does 7 damage].
## Why / Balance
As of right now, the Svalinn Laser pistol is, in almost every way outclassed by any other ranged weapon. While the ability for an energy weapon to reload its power source is interesting and useful, dealing a pitiful 7 damage per shot means that it would require a full medium-capacity power cell plus a few extra shots to deal a minimum of 100 damage. This makes the Svalinn generally unviable for just about every purpose, since a weapon that cannot deal reasonable damage just does not compare to the much stronger offerings of other weapons.

While this may somewhat intrude on the viability of the Advanced and Antique laser pistols, I believe that the need to carry extra power cells as ammunition and still slightly inferior damage would keep it in check, while still making it viable as a weapon. The inability to refill batteries on the spot without a tier 3 technology would make the Svalinn weak for extended work involving combat, and the overall damage of the Svalinn is still inferior to a decent amount of ballistic weapons. These changes should still give room for ballistic weapons and true infinite ammo laser weapons to still have their distinct advantages and disadvantages over this weapon.
## How to test
1. Start a local instance
2. Spawn in a Svalinn Laser Pistol and a target.
3. Shoot a target and check the damage number

## Changelog
- tweak: changed the power cell projectile type to RedLaser.
